src/transformers/file_utils.py

```diff
@@ -2083,8 +2159,7 @@ def is_torch_fx_proxy(x):
 
 def is_tensor(x):
     """
-    Tests if ``x`` is a :obj:`torch.Tensor`, :obj:`tf.Tensor`, obj:`jaxlib.xla_extension.DeviceArray` or
-    :obj:`np.ndarray`.
+    Tests if `x` is a `torch.Tensor`, `tf.Tensor`, obj:*jaxlib.xla_extension.DeviceArray* or `np.ndarray`.
```

Comment:
I see an :obj: remaining here!

Reply:
It didn't have it's proper double columns. Thanks for flagging!